### DISCUSSION TITLE
Is there any place that documents the annotations exposed by Spring Modulith?

### DISCUSSION THREAD

**Discussion by nagkumar** [spring-projects/spring-modulith#233] [General]
I believe spring modulith does the magic of bringing in simple way to control the modularity of code with these annotations

1. ApplicationModule
2. ApplicationModuleInitializer
3. ApplicationModuleListener
4. Modulith
5. Modulithic
6. NamedInterface

and for test code

1. ApplicationModuleTest
2. EnableScenarios

Any good documentation and if possible nice working examples that can give expertise on how to use them correctly..

I do see spring-modulith examples do contain ApplicationModule, ApplicationModuleListener, and NamedInterface that way partially covered use of these.. however unable to find any code that talks about the rest.

**Comment by odrotbohm** [comment]
Did you check out the [reference documentation](https://docs.spring.io/spring-modulith/docs/current/reference/html/) and [Javadoc](https://docs.spring.io/spring-modulith/docs/current/api/)?
